sdc/hiframes/pd_dataframe_ext.py

```diff
@@ -59,6 +59,32 @@ class ColumnLoc(NamedTuple):
     col_id: int
 
 
+def get_columns_loc(col_types, col_names):
```

Comment:
Let's rename the function. Maybe `get_structure_maps`?

Reply:
done